Assets/Scripts/Utilities/WadeUtils.cs

```diff
@@ -402,6 +404,36 @@ public static GameObject TempInstantiate(GameObject obj, Vector3 pos, Quaternion
 		return go;
 	}
 
+	public static T GetCopyOf<T>(this Component comp, T other) where T : Component
```

Comment:
Interesting. Do you have a source on mobile having problems w/ it? I'd like to read more about it.